src/onegov/town6/assets/js/file-table-row-toggler.js

```diff
@@ -0,0 +1,24 @@
+$(document).ready(function () {
+    // Handle click on the button to toggle the detail row
+    $('a[id^="button-"]').on('click', function (e) {
+        e.preventDefault();
+
+        // Get the corresponding file number and detail row
+        var fileNumber = $(this).attr('id').split('-')[1];
+        var detailRow = $('#details-' + fileNumber);
+
+        // Close all other detail rows
+        $('tr[id^="details-"]').not(detailRow).addClass('hidden');
```

Comment:
Personally, I wouldn't close the other detail rows and leave it to the user to close it. Like this one could compare to files..

Reply:
I thought about that too, but does one actually ever close a the details tab if the file isn't the right one?